pkg/util/pki/pki_common_test.go

```diff
@@ -175,3 +175,42 @@ func TestControllerUserForCluster(t *testing.T) {
 		t.Errorf("Expected %+v\nGot %+v", expected, user)
 	}
 }
+
+func TestTruncatedCommonName(t *testing.T) {
+	testCases := []struct {
+		testName             string
+		commonName           string
+		expectedHashedString string
+		expectedLen          int
+	}{
+		{
+			testName:    "Common name with length exceeded limitation",
+			commonName:  "kafka-test-controller-exceeded-length.test-namespace.mgt.cluster.local",
+			expectedLen: MaxCNLen,
```

Comment:
beside the lenght also the expected output could be verified as well.

Reply:
yea...but we since don't actually care about the content of the hashed string, I don't see the necessity of verifying it in the test